src/ServiceControlInstaller.Engine/Instances/Transports.cs

```diff
@@ -17,30 +17,30 @@ public static TransportInfo FindByName(string name)
             {
                 Name = "AzureServiceBus",
                 TypeName = "NServiceBus.AzureServiceBusTransport, NServiceBus.Azure.Transports.WindowsAzureServiceBus",
-                MatchOn = "NServiceBus.AzureServiceBusTransport",
+                MatchOn = "NServiceBus.AzureServiceBus,",
```

Comment:
@johnsimons I think you made a typo in all the MatchOn values you put back - each one has a comma at the end